frontend/integration-tests/tests/olm/global-installmode.scenario.ts

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable no-undef, no-unused-vars */
+
+import { browser, $, element, ExpectedConditions as until, by } from 'protractor';
+import { safeDump, safeLoad } from 'js-yaml';
+import { defaultsDeep } from 'lodash';
+import { execSync } from 'child_process';
+
+import { appHost, testName, checkLogs, checkErrors } from '../../protractor.conf';
+import * as crudView from '../../views/crud.view';
+import * as catalogView from '../../views/olm-catalog.view';
+import * as sidenavView from '../../views/sidenav.view';
+import * as yamlView from '../../views/yaml.view';
+
+describe('Interacting with the Redis Operator (all-namespaces install mode)', () => {
```

Comment:
The etcd Operator we are shipping in upstream-community-operators` doesn't support `AllNamespaces` install mode, so using Redis Operator instead.

Reply:
In the future, we should be testing against a static e2e quay namespace with a dummy operator, to avoid potential issues when community-operators get updated